[FIX] elastic_service: Add support for 'in' operator into scoped queries and allows raw_es_query

lmignon opened the pull request:
```
{%with_scope raw_es_query: {filter:{terms:{attributes.color:['red', 'yellow']}}} %}
```

lmignon committed:
[FIX] elastic_service: Add support for 'in' operator into scoped queries

rvalyi commented on lib/shop_invader/services/elastic_service.rb:
```diff
@@ -142,6 +142,8 @@ def build_params(conditions)
               end
             elsif %w(gt gte lt lte).include?(op)
               params[:bool][:filter] << { range: { name => { op => value } } }
+            elsif %w(in).include?(op)
```
not a big deal but %w is to make list of strings. For a single string IMHO it is cleaner to have
`elsif op == "in"`

lmignon replied:
@rvalyi I'm not a ruby developer :smirk: I'll change the code

lmignon replied:
@rvalyi done

lmignon committed:
[IMP] elastic_service: Add support for raw queries into scoped queries
if the sope containes raw_es_query, the value is used as query when calling elastisearch server

sebastienbeau reviewed (approved):
Ok for me thanks the fix and the improvement, it's great to be able to do directly "raw" request
